python/pycylon/pycylon/api/lib.pyx

```diff
@@ -190,3 +194,35 @@ cdef api object pycylon_wrap_base_arrow_index(const shared_ptr[CBaseArrowIndex]
     cdef BaseArrowIndex base_arrow_index = BaseArrowIndex.__new__(BaseArrowIndex)
     base_arrow_index.init(cbase_arrow_index)
     return base_arrow_index
+
+cdef api bint pyclon_is_column(object column):
+    return isinstance(column, Column)
+
+cdef api shared_ptr[CColumn] pycylon_unwrap_column(object column):
+    cdef Column col
+    if pyclon_is_column(column):
+        col = <Column> column
+        return col.thisPtr
+    else:
+        raise ValueError('Passed object is not a Cylon Column')
+
+cdef api object pycylon_wrap_column(const shared_ptr[CColumn]& ccol):
+    cdef Column col = Column()
+    col.init(ccol)
+    return col
+
+cdef api bint pyclon_is_scalar(object scalar):
+    return isinstance(scalar, scalar)
```

Comment:
should this be 

`return isinstance(scalar, Scalar)` instead

Reply:
oh yes! :D